.changeset/gentle-meals-crash.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': major
+---
+
+Remove support for `Astro.__renderMarkdown` which is used by `@astrojs/markdown-component`. The package has been deprecated since Astro v1. To continue using this feature in Astro v3, you can use community packages like https://github.com/natemoo-re/astro-remote instead.
```

Comment:
```suggestion
Remove support for `Astro.__renderMarkdown` which is used by `@astrojs/markdown-component`. 

The `<Markdown />` component was deprecated in Astro v1 and is completely removed in v3. This integration must now be removed from your project.

As an alternative, you can use community packages that provide a similar component like https://github.com/natemoo-re/astro-remote instead.
```
Just emphasizing that the users will probably think more about the actual component than the package, so wanted to make sure that was explicit. Something like this?

Reply:
Sounds great. I think it flows better too.